components/colorarea/metadata/colorarea.yml

```diff
@@ -53,10 +56,10 @@ examples:
   - id: color-area
     name: Custom Size
     markup: |
-      <div class="spectrum-ColorArea" style="width: 72px; height: 72px">
+      <div class="spectrum-ColorArea" style="width: 80px; height: 80px">
```

Comment:
should this be changed to  `"--mod-colorarea-height: 80px; --mod-colorarea-width: 80px;",`